### PR TITLE
Danielle/update-sdk 7

### PR DESCRIPTION
**Pull request by daniellepintz**

### CODE REVIEW

**Review by tcnguyen — APPROVED** [review]

File: src/dialogs/search-dialog.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ class SearchDialog extends PromptDialog {
     const facet = missingEntities.keys().next().value;
     const facetValueCounts = (await this.db.getValuesByFacet([facet], this.query))[facet];
     return {
-      facetValueCounts,
+      facetValueCounts
```
Comment (yangeorget) [inline src/dialogs/search-dialog.js]: does this pass the style checking?
usually we put a comma

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "release": "yarn test && yarn publish --conventional-commits --changelog-preset=angular"
   },
   "peerDependencies": {
-    "botfuel-dialog": "5.x"
+    "botfuel-dialog": "7.x"
```
Comment (michelbl) [inline package.json]: Do we provide a version for users of botfuel-dialog 6.x?

**Review by michelbl — CHANGES_REQUESTED** [review]

File: src/dialogs/search-dialog.js (hunk 1 above)
Comment (daniellepintz) [reply src/dialogs/search-dialog.js]: How do I run style checking? Will revert to previous version.

Comment (michelbl) [reply src/dialogs/search-dialog.js]: `yarn run style`

Could you add CI with tests and linting? (see https://github.com/Botfuel/botfuel-dialog/blob/master/.circleci/config.yml, I can help you if you want)

File: package.json (hunk 2 above)
Comment (daniellepintz) [reply package.json]: Ah, right.  So I should deploy the module two times, the first time with the dependency: ">=5 <7" and the second time with the dependency "7.x"?

File: src/dialogs/search-dialog.js (hunk 1 above)
Comment (daniellepintz) [reply src/dialogs/search-dialog.js]: I don't think there are any style tests... Yes, I can do that, and will let you know if I need help.

File: package.json (hunk 2 above)
Comment (michelbl) [reply package.json]: It seems to me that in this PR you fixes changes that were introduced by 6 *and* 7. I would split that work into 2 PRs, one for 5->6 and one for 6->7.
This in not a lot of work using git tools like kraken. Ask me if you want.

File: src/dialogs/search-dialog.js (hunk 1 above)
Comment (michelbl) [reply src/dialogs/search-dialog.js]: There are linting and tests: 
```
      - run: yarn run style
      - run: yarn test
```

File: package.json (hunk 2 above)
Comment (tcnguyen) [reply package.json]: is it neccessary?

**Review by michelbl — APPROVED** [review]

**Pull request merged**